src/scripts/Components/TabPanel/Reuse/ContentList/ContentItemTabular.scss

```diff
@@ -59,9 +59,13 @@
   .h5p-hub-owner {
     font-size: 1em;
 
-      & > a {
-          color: #326efb;
+    & > a {
+      color: #326efb;
+
+      &:focus {
+        outline: 5px auto -webkit-focus-ring-color;
```

Comment:
`-webkit-focus-ring-color` is not a valid color, suggest omitting it to use the browser default